IntroComponentWithSign-upForm/script.js

```diff
@@ -0,0 +1,67 @@
+const form  = document.getElementById("form")
+const firstName  = document.getElementById("firstName")
+const lastName  = document.getElementById("lastName")
+const email  = document.getElementById("email")
+const password  = document.getElementById("password")
+//show error message
+const showErrorMessage = ( input , message ) => {
+    const span = document.querySelector("."+input.id)  
```

Comment:
Add spaces for better reading!

```suggestion
    const span = document.querySelector("." + input.id)  
```